src/astro/files/locations/http.py

```diff
@@ -0,0 +1,18 @@
+from typing import Dict, List, Union
+
+from astro.constants import FileLocation
+from astro.files.locations.base import Location
+
+
+class Http(Location):
+    """Handler http location operations"""
+
+    location_type = FileLocation.HTTP
+
+    def get_paths(self) -> List[str]:
+        """Resolve patterns in path"""
+        return [self.path]
+
+    def get_transport_params(self) -> Union[Dict, None]:
+        """Dummy method"""
+        return None
```

Comment:
Why is this a dummy method, is this because transport params are not required for HTTP, in that case can we add a comment about it

Reply:
or `return None` ins BaseLocation -- no strong preference between them

Reply:
Done.